### PR TITLE
Reprocess Stripe events

### PR DESCRIPTION
**Pull request by ErikSchierboom**
According to the docs:

> List events, going back up to 30 days.

### CODE REVIEW

File: app/commands/payments/stripe/handle_event.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Payments::Stripe::HandleEvent
```
Comment (ErikSchierboom) [inline app/commands/payments/stripe/handle_event.rb:1]: I've extracted this bit of logic to be re-usable

Comment (iHiD) [reply app/commands/payments/stripe/handle_event.rb:1]: Nice.

File: app/commands/payments/stripe/payment/sync_all.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Payments::Stripe::Payment::SyncAll
+  include Mandate
+
+  def call
+    Stripe::Event.list({ limit: 100, type: 'payment_intent.succeeded' }).auto_paging_each do |event|
+      next if existing_payment_ids.include?(event.data.object.id)
```
Comment (iHiD) [inline app/commands/payments/stripe/payment/sync_all.rb:6]: Clever. I like this.

**Review by iHiD — APPROVED** [review]
I haven't merged as it's still a draft, but looks great to me.

**Comment by ErikSchierboom** [comment]
> I haven't merged as it's still a draft, but looks great to me.

I wanted you to have a peek first to see if you liked the approach. Feel free to merge

**Pull request merged**